src/TokenKind.php

```diff
@@ -170,6 +170,8 @@ class TokenKind {
     const StringReservedWord = 320;
     const BoolReservedWord = 321;
     const NullReservedWord = 322;
+    const MixedReservedWord = 420;
```

Comment:
```suggestion
    const MixedReservedWord = 340;
```

It seems like `// const DollarOpenCurly = 339;` was used in the past, and all of the other reserved words except for iterable were using 330, and 400 is for special kinds of tokens?

The values don't affect the behavior of the parser, and I didn't notice this when contributing or reviewing (iterable) in the past (e.g. https://github.com/microsoft/tolerant-php-parser/pull/245), but consistently using 3xx for reserved words as much as possible would be preferable